CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and **Merged pull requests**. Critical items to know are:
 Referenced versions in headers are tagged on Github, in parentheses are for pypi.
 
 ## [vxx](https://github.com/urlstechie/urlschecker-python/tree/master) (master)
+ - adding support for dotfiles for a file type (0.0.21)
```

Comment:
maybe just say for dotfiles.
_for dotfiles for a file type_ is a bit confusing.

Reply:
for the file type is in reference to the parameter. It's just the changelog, so I think more information is better.